generatebundlefile/README.md

```diff
@@ -41,10 +41,132 @@ generatebundlefile --generate-sample
 
 This will output a sample bundle file to ./output.
 
+
+### Public Promotion to another Account
+
+```sh
+generatebundlefile --private-profile "profile-name" --input data/sample_input.yaml
+```
+
+This command will move **Only** the helm chart from the listed input files to the target **public** ECR in another account.
```

Comment:
```suggestion
The command above will move **Only** the helm chart from the listed input files to the target **public** ECR in another account.
```

Best to be more explicit, since there are commands both above and below this statement, and it's not immediately obvious to which command it is referring.